swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        if self.condition == 0:
+            return "It's giving 'Go home, Roger.' Don't do it. "
+        elif self.condition == 1:
+            return "Get this only if it's free."
+        elif self.condition == 2:
```

Comment:
Our tests don't check the descriptions, but it does set the condition to 3.5. How would this function change if the Item's condition was a float value? We should always consider how other numeric types (like floats) would be processed in our code.